plugins/dominant-color-images/hooks.php

```diff
@@ -95,6 +95,12 @@ function dominant_color_img_tag_add_dominant_color( $filtered_image, string $con
 		$filtered_image = '';
 	}
 
+	$processor = new WP_HTML_Tag_Processor( $filtered_image );
+
+	if ( ! $processor->next_tag( array( 'tag_name' => 'IMG' ) ) ) {
+		return $filtered_image;
+	}
+
```

Comment:
This condition should be moved below the `if ( 'the_content' !== $context ) {` condition below.